fili-core/src/main/java/com/yahoo/bard/webservice/config/BardFeatureFlag.java

```diff
@@ -7,8 +7,9 @@
  */
 public enum BardFeatureFlag implements FeatureFlag {
     PARTIAL_DATA("partial_data_enabled"),
-    DRUID_CACHE("druid_cache_enabled"),
+    @Deprecated DRUID_CACHE("druid_cache_enabled"),
     DRUID_CACHE_V2("druid_cache_v2_enabled"),
```

Comment:
Per conversation yesterday, the values for the new caching strategy config param:
- ETag
- LocalSignature
- TtlOnly
- None